pkg/kube/wait.go

```diff
@@ -259,3 +257,16 @@ func getPods(client kubernetes.Interface, namespace string, selector map[string]
 	})
 	return list.Items, err
 }
+
+func isPodReady(pod *v1.Pod) (ready bool) {
+	if &pod.Status != nil && len(pod.Status.Conditions) > 0 {
+		for _, condition := range pod.Status.Conditions {
+			if condition.Type == v1.PodReady &&
+				condition.Status == v1.ConditionTrue {
+				ready = true
```

Comment:
This should probably just return `true` without the named param. No need for a `break`